Godeps/Godeps.json

```diff
@@ -3929,11 +3929,6 @@
 			"Rev": "225eecce40065518f2eabc3b52020365db012384"
 		},
 		{
-			"ImportPath": "k8s.io/kubernetes/cmd/libs/go2idl/args",
```

Comment:
I wonder why commit-checker doesn't complain about the title of this commit, when it's modifying `Godeps.json`.